app/views/reviews/new.html.erb

```diff
@@ -1,5 +1,8 @@
 <h2>Add New Review</h2>
 <section>
+  <% flash.each do |name, msg| %>
+    <div class="error_message"><%= msg %></div>
```

Comment:
We should get some more clarification on how to nest these in the application.html template during check-in